src/todopage/ToDo.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+
+library.add(faTrash);
+
+const DeleteIcon = styled(FontAwesomeIcon)``;
+
+function getColor(status) {
+  if (status === 'active') {
+    return 'red';
+  } else if (status === 'inProgress') {
+    return 'yellow';
+  } else {
+    return 'green';
+  }
+}
+
+function getNewStatus(status) {
+  if (status === 'active') {
+    return 'inProgress';
+  } else if (status === 'inProgress') {
+    return 'complete';
+  } else if (status === 'complete') {
+    return 'active';
+  } else {
+    return 'active';
+  }
+}
+
+const ToDoContainer = styled.div`
```

Comment:
you can think about using a `section` instead of a `div` to be more specific? so every todo is a section itself